lib/seamless_database_pool/controller_filter.rb

```diff
@@ -6,35 +6,28 @@ module SeamlessDatabasePool
   # when you need different connection types.
   #
   # Example:
-  # 
+  #
   #   ApplicationController < ActionController::Base
   #     include SeamlessDatabasePool::ControllerFilter
   #     use_database_pool :all => :persistent, [:save, :delete] => :master
   #     ...
-  
-  module ControllerFilter    
+  module ControllerFilter
     def self.included(base)
       unless base.respond_to?(:use_database_pool)
         base.extend(ClassMethods)
         base.class_eval do
-          if base.method_defined?(:perform_action) || base.private_method_defined?(:perform_action)
```

Comment:
my first try on changing this has been creating a different module per every action (https://github.com/bdurand/seamless_database_pool/pull/37/commits/f402c59bdffbc1ababea024fba49420e6b4c3348) Eventually I decided to join them all in one common module https://github.com/bdurand/seamless_database_pool/pull/37/commits/4ca37c18296dbee70df687b2eb9b19f6893a3ebb

I like both options but just let me know what you prefer ✌️